build.xml

```diff
@@ -9,39 +9,50 @@
   <property name="skip.tests" value="true"/>
    -->
 
-	<property name="version" value="0.0.0" />
+	<exec executable="git" outputproperty="git.revision">
+		<arg value="describe" />
+		<arg value="--always" />
+	</exec>
+	<property name="version" value="0.0.0-${git.revision}" />
```

Comment:
I don't think I follow what you mean. You mean like the `_versionXXX.xml` files?